src/PROPOSAL/PROPOSAL/math/RandomGenerator.h

```diff
@@ -32,7 +32,7 @@
 #include <functional>
 #include <random>
 #include <iostream>
-
+#include "PROPOSAL/secondaries/parametrization/bremsstrahlung/BremsKochMotz.h"
```

Comment:
Why does this needs to be included here?

Reply:
The sampling method of Koch and Motz needs an amount of random numbers that is not previously known when calling the `CalculateDirections` function. Therefore, I use one random number as a seed to create new random numbers inside the BremsKochMotz class.

However, the RandomGenerator  of PROPOSAL is written in such a way that there is only one global random generator object. In order to not interfere with the global random generator, the BremsKochMotz class creates an own RandomGenerator. Since